src/pages/Profile/index.jsx

```diff
@@ -14,17 +15,19 @@ import styles from './index.scss';
 }))
 @observer
 export default class Profile extends Component {
-  static async onEnter({ userStore }) {
-    await userStore.fetchDetail();
-  }
-
   constructor(props) {
     super(props);
     this.state = {
       currentForm: 'basic'
     };
   }
 
+  componentDidMount() {
```

Comment:
put this logic in `onEnter`, or SSR will not work

Reply:
If put in onEnter ，refresh page getCookie method will error .